### PR TITLE
Some changes from a first-timer.

### PR DESCRIPTION
**Pull request by axyjo**
Here are a few things about CCs and SSNs.

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -150,12 +151,14 @@ Something to look into.
 Sometimes you can get BoA or TD America to look up your Canadian credit history.
 Really tough if you are under 21.
 
+One surefire way of getting a credit card fairly soon would be to apply for a Capital One credit card that's directly targeted for people with no credit histroy/newcomers. Beware this card is free for the first year, but costs around $30 per year after that.
```
Comment (sholiday) [inline README.markdown:154]: Were you able to successfully get this as a non resident alien? We're you over 21?

**Comment by sholiday** [comment]
Awesome stuff! Thanks so much. My only questions are about the credit card info.

File: README.markdown (hunk 1 above)
Comment (axyjo) [inline README.markdown:154]: Nope. I'm only 18 and I got it.

**Comment by sholiday** [comment]
Merged, thanks!

**Pull request closed without merging**